WBooks/Models/Book.swift

```diff
@@ -31,6 +31,6 @@ extension Book: Argo.Decodable {
     }
 }
 
-internal func toURL(urlString: String?) -> Decoded<URL?> {
+fileprivate func toURL(urlString: String?) -> Decoded<URL?> {
     return Decoded<URL?>.fromOptional(URL(string: urlString ?? ""))
```

Comment:
This should be `urlString.flatMap { URL(string: $0) }`.
Because if there is no string, we want no url. And I'm not sure, but I think `URL(string: "")` gives you a URL instead of failing and giving you .none.